### PR TITLE
Add some dialogs in sponsors for feedbacks and advices

### PR DESCRIPTION
**Pull request by JoseFcoEstrella**
#### Description:
In this ticket have some changes:
- New sponsor is added
- A dialog in sponsors section is added (To inform the user about "why it takes long time to loading the sponsors")
- Another dialog for feedbacks when sponsors dont show something or take long time in loading

Dialog content:
- **First dialog in sponsor:** Some of the sponsors take a moment to load due to that sponsor's page
- **When page take long time in load:** It is taking longer than expected, please wait a moment or check your internet connection.
- **When have a error in load:** An error occurred during loading. Please try again later.
#### Linked issues:

[#166](https://trello.com/c/zKpiqAO3/166-actualizar-dos-url-en-auspiciadores)

#### Screenshots:

Dialog in sponsors section

new sponsor

#### How to test:

In app-->Go in settings--> sponsonsors and testing dialogs

### CODE REVIEW

File: app/locales/es.json [hunk 1]
```diff
@@ -77,6 +77,9 @@
     "current_situation_label": "Situación actual",
     "default_news_site_name": "Noticias de Safe Paths",
     "deceased_label": "Fallecidos",
+    "dialog_advice": "Algunos de los patrocinadores tardan un momento en cargar debido a la página de dicho patrocinador",
```
Comment (inteljudajocu) [inline app/locales/es.json]:
```suggestion
    "dialog_advice": "Algunos de los patrocinadores pueden tardan un momento en cargar debido a la página de dicho patrocinador",
```

File: app/locales/es.json [hunk 2]
```diff
@@ -77,6 +77,9 @@
...
+    "dialog_error_advice": "Se produjo un error durante la carga. Por favor intente nuevamente más tarde.",
```
Comment (inteljudajocu) [inline app/locales/es.json]:
```suggestion
    "dialog_error_advice": "Se produjo un error durante la carga. Por favor inténtelo nuevamente más tarde.",
```

File: app/views/DR/News/Details.js [hunk 3]
```diff
@@ -58,6 +95,21 @@ const Details = ({
           source={source}
           startInLoadingState
           injectedJavaScript={fixerImage}
+          onLoadStart={() => {
+            setNewTimer(
+              setTimeout(() => {
+                setDialogText(t('label.dialog_interval_advice'));
+                setShowDialog(true);
+              }, 15000),
```
Comment (inteljudajocu) [inline app/views/DR/News/Details.js]: Decrease timer to  10000

**Review by inteljudajocu — CHANGES_REQUESTED** [review]
The changes in wording should be applied to other languages.

**Review by inteljudajocu — APPROVED** [review]

**Comment by D-Rosa99** [comment]
This piece of code appear in two places (***app/views/DR/Sponsors/index.js***, ***app/views/DR/News/Details.js***), and the only different is the text (***the one with a red square***), I think you can put this in a separate file and just call it wherever you want it to use and pass the text to show

Let me know what you think :+1: 

image

**Review by RutsSantos — APPROVED** [review]
LGTM

**Review by Faustrox — APPROVED** [review]
LGTM

**Review by Faustrox — APPROVED** [review]
LGTM

**Review by D-Rosa99 — APPROVED** [review]

**Pull request merged**